internal/cli/root/root.go

```diff
@@ -43,6 +43,7 @@ func init() {
 	region := Cmd.Flag("region", "Override the region.").Short('r').String()
 	workdir := Cmd.Flag("chdir", "Change working directory.").Default(".").Short('C').String()
 	verbose := Cmd.Flag("verbose", "Enable verbose log output.").Short('v').Bool()
+	configFile := Cmd.Flag("file", "Override the config file").Default("up.json").Short('d').String()
```

Comment:
Just `--config` for now with no short is probably fine, I imagine it wont get too much use, maybe not worth hogging a short flag